lib/basedriver/capabilities.js

```diff
@@ -11,7 +11,7 @@ function mergeCaps (primary = {}, secondary = {}) {
   for (let [name, value] of _.toPairs(secondary)) {
     // Overwriting is not allowed. Primary and secondary must have different properties (w3c rule 4.4)
     if (!_.isUndefined(primary[name])) {
-      throw new Error(`property ${name} should not exist on both primary and secondary`);
+      throw new Error(`property '${name}' should not exist on both primary and secondary`);
```

Comment:
I think we can improve his error message by actually displaying both primary and secondary objects via JSON.stringify